script/sdk-js-script-tags.hbs

```diff
@@ -0,0 +1,27 @@
+{{!--
```

Comment:
What would you think of abstracting out the `sdkUrl` and `localePrefix` partials somewhere so we could also use them when generating the compiled templates script tags?

Reply:
yes that's a great idea! I updated the theme's test hbs instance so that we can test partial usages like this. I think long term we'll want jambo to be usable directly in javascript instead of only via cli, which would alleviate the need to keep this "mock jambo" around.